src/DataLakeStore/DataLakeStore.Test/ScenarioTests/AdlsTests.ps1

```diff
@@ -857,10 +856,11 @@ function Test-DataLakeStoreFileSystemPermissions
 		Assert-AreEqual $($currentCount+1) $result.Count
 
 		# Export Acl
-		$targetFile = "/aclOutputFile"
-		Export-AzDataLakeStoreChildItemProperties -Account $accountName -Path "/" -OutputPath $targetFile -GetAcl -IncludeFile -SaveToAdl
-		$result = Get-AzDataLakeStoreItem -Account $accountName -path $targetFile
+		$targetFile = "./ScenarioTests/acloutput"
```

Comment:
same comment

Reply:
Assuming this specifies a local path an not an ADLS path

Reply:
fixed.